packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -77,28 +81,72 @@ const Dashboard = ({
     bannerProps === FAILED
       ? errorBannerProps
       : (bannerProps as { bannerHeading: string; isSuccess: boolean });
+
+  const schemesUserOwnsIsEmpty = schemesUserOwns.length === 0;
+  const schemesUserCanEditIsEmpty = schemesUserCanEdit.length === 0;
+
   return (
     <>
       {isTechSupportUser && <Navigation isSuperAdminNav={false}></Navigation>}
       <div className="govuk-grid-row govuk-!-padding-top-7">
         <Meta title="Dashboard - Manage a grant" />
-        <div className="govuk-grid-column-two-thirds govuk-!-margin-bottom-6">
+        <div className="govuk-grid-column-full govuk-!-margin-bottom-6">
           {bannerProps && <ImportantBanner {...formattedBannerProps} />}
           <AccountDetails userDetails={userDetails} />
-          <ManageGrantSchemes schemes={schemes} />
-          <CustomLink
-            href="/new-scheme/name"
-            isButton
-            dataCy="cy_addAGrantButton"
-          >
-            Add a grant
-          </CustomLink>
+
+          {schemesUserOwnsIsEmpty && schemesUserCanEditIsEmpty
+            ? noGrantsView()
+            : grantsView(
+                schemesUserOwns,
+                schemesUserCanEdit,
+                schemesUserOwnsIsEmpty,
+                schemesUserCanEditIsEmpty
+              )}
```

Comment:
this works, but it's not great practice, we should use jsx syntax here instead of calling the function. (`noGrantsView` & `grantsView` are react components because they return jsx)

eg: `<NoGrantsView />`

Reply:
sorted